legacy/src/api/assets/asset-transfers.md

```diff
@@ -281,3 +282,37 @@ Returns a [paginated][] list of Asset Transfers.
 
 [Quota Error Response]: {% link api/quotas.md %}#quota-error-response
 [paginated]: {% link api/pagination.md %}
+
+## Cancel an Asset Transfer
+{% reqspec %}
+  POST '/api/asset-transfers/{assetTransferId}/cancel'
+  auth 'api-key'
+  path_param 'assetTransferId', 'M7Kn2stAxNa6ri7h'
+{% endreqspec %}
+
+{% h4 Example response payload %}
+
+{% json %}
+{
+  "id": "M7Kn2stAxNa6ri7h",
+  "status": "cancelled",
+  "assetId": "YGRo6TYYSxH3js7",
+	"value": "6000",
+  "description": "$60 Giftcard",
+  "message": "Happy birthday",
+  "senderName": "My Cafe",
+  "senderAccountId": "aBc932S9182qwCDqwer",
+  "recipientAccountId": "9EDxUT91TMsUjoqoQeBuLQ",
+	"recipientAlias": "+64*****2345",
+  "createdAt": "2020-05-01T12:30:00.000Z",
+  "updatedAt": "2020-05-02T01:03:37.222Z",
+  "suppressNotifications": false
+}
+{% endjson %}
+
+{% h4 Error Responses %}
+
+| Status |             Code              |                               Description                               |
+| :----- | :---------------------------- | :---------------------------------------------------------------------- |
+| 403    | FORBIDDEN                     | The asset transfer has not been found or there are missing permissions. |
+| 403    | INVALID_ASSET_TRANSFER_STATUS | The asset transfer status is invalid and cannot be cancelled.           |
```

Comment:
I think a better description here would be

"The asset transfer is in a final status and cannot be canceled."

Reply:
@Neeraj-Ghodla Expired asset transfers cannot be cancelled. Expired is not a final status so this isn't necessarily true.